### PR TITLE
refactor(ext/cron): align error messages

### PR DESCRIPTION
**Pull request by irbull**
Aligns the error messages in the cron extension to be in-line with the Deno style guide.

https://github.com/denoland/deno/issues/25269

### CODE REVIEW

File: ext/cron/01_cron.ts [hunk 1]
```diff
@@ -103,10 +105,14 @@ function cron(
   handler2?: () => Promise<void> | void,
 ) {
   if (name === undefined) {
-    throw new TypeError("Deno.cron requires a unique name");
+    throw new TypeError(
+      "Cannot create cron job, a unique name is required: received `undefined`",
```
Comment (lucacasonato) [inline ext/cron/01_cron.ts:109]:
```suggestion
      "Cannot create cron job, a unique name is required: received 'undefined'",
```

File: ext/cron/01_cron.ts [hunk 2]
```diff
@@ -103,10 +105,14 @@ function cron(
...
+    );
   }
   if (schedule === undefined) {
-    throw new TypeError("Deno.cron requires a valid schedule");
+    throw new TypeError(
+      "Cannot create cron job, a schedule is required: received `undefined`",
```
Comment (lucacasonato) [inline ext/cron/01_cron.ts:114]:
```suggestion
      "Cannot create cron job, a schedule is required: received 'undefined'",
```

File: ext/cron/01_cron.ts (hunk 1 above)
Comment (irbull) [reply ext/cron/01_cron.ts:109]: Good catch. I'm going to try and use the single ticks everywhere. Maybe we should add that to the style guide too.

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thanks!

**Pull request merged**